.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "docker"]
+	path = docker
+	url = git@github.com:spryker/docker-sdk.git
```

Comment:
@sprymiker Why is this now part of the public repo? I thought the whole idea of this is to avoid this going into this public suite?

Reply:
@dereuromark The idea is not to avoid to have it public. The idea to ship Docker SDK separately. So we are ok to have submodule in `spryker-shop/suite`, as you said it is not possible to remove it during releasing.

Reply:
We can remove it, but then it would be removed for good.
Including via composer is not an option, you said?

Reply:
Composer is not an option, right.

If we can remove it from split and resolve conflicts during releasing on the split side to keep it always deleted. let's go for it -  I am ok.

Reply:
Feel free to open Pr